torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -202,6 +252,12 @@ class LinkNeighborLoader(torch.utils.data.DataLoader):
             which neighbors are sampled. Must be the same length as
             the :obj:`edge_label_index`. If set to :obj:`None` its set to
             `torch.zeros(...)` internally. (default: :obj:`None`)
+        edge_time (Tensor, optional): The timestamps for edge indicies for
+            which neighbors are sampled. Must be the same length as
+            the :obj:`edge_label_index`. If set, temporal sampling will be
+            used such that neighbors are guaranteed to fulfill temporal
+            constraints, *i.e.*, neighbors have an earlier timestamp than
+            the ouput edge. (default: :obj:`None`)
```

Comment:
Can we add some documentation here that `time_attr` needs to be set for this as well?